src/App.tsx

```diff
@@ -31,45 +35,120 @@ function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
-  // ...
+  switch (sortType) {
+    case SortType.ALPABET:
+      visibleGoods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SortType.LENGTH:
+      visibleGoods.sort((a, b) => a.length - b.length);
+      break;
+    case SortType.NONE:
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      visibleGoods;
+      break;
+
+    default:
+      throw new Error(`Case ${sortType} not found`);
```

Comment:
Do you really need this error? Why can't You return the original array without sorting?